### PR TITLE
Add Splash Screen, Add dimen and integer files

### PR DESCRIPTION
**Pull request by oscar-amezquita**
## Description
- Create Splash Screen Activity.
- add image resources (GitHub logo).
- Add resources files (Dimen and integer).
- Add Splash Screen with no action bar.

## Evidence
capture

### CODE REVIEW

File: app/src/main/java/com/globant/example/mentorapp/view/activities/SplashScreenActivity.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+        TimerTask task = new TimerTask() {
+            @Override
+            public void run() {
+
+
+            }
+        };
+        Timer timer = new Timer();
+        timer.schedule(task, R.integer.splash_screen_duration);
```
Comment (daniellemariani) [inline app/src/main/java/com/globant/example/mentorapp/view/activities/SplashScreenActivity.java]: Keep in mind you're passing a resource id and not the value. You should use `getResources#getInteger()`

Double check this.

File: build.gradle [hunk 2]
```diff
@@ -15,6 +15,7 @@ buildscript {
 allprojects {
     repositories {
         jcenter()
+        maven { url "https://jitpack.io" }
```
Comment (daniellemariani) [inline build.gradle]: Do you really need this?

File: app/src/main/java/com/globant/example/mentorapp/view/activities/SplashScreenActivity.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+import java.util.Timer;
+import java.util.TimerTask;
+
+public class SplashScreenActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        // Set portrait orientation
+        setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
```
Comment (daniellemariani) [inline app/src/main/java/com/globant/example/mentorapp/view/activities/SplashScreenActivity.java]: One of the object goals is to support both Portrait and Landscape

File: app/src/main/java/com/globant/example/mentorapp/view/activities/SplashScreenActivity.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
+package com.globant.example.mentorapp.view.activities;
+
+import android.content.Intent;
+import android.content.pm.ActivityInfo;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.Window;
```
Comment (daniellemariani) [inline app/src/main/java/com/globant/example/mentorapp/view/activities/SplashScreenActivity.java]: I think this import is not necessary

**Review by daniellemariani — COMMENTED** [review]
Also attach a screenshot or gif

File: app/src/main/java/com/globant/example/mentorapp/view/activities/SplashScreenActivity.java (hunk 4 above)
Comment (oscar-amezquita) [reply app/src/main/java/com/globant/example/mentorapp/view/activities/SplashScreenActivity.java]: ok, clear unnecessary imports, done.

File: app/src/main/java/com/globant/example/mentorapp/view/activities/SplashScreenActivity.java (hunk 3 above)
Comment (oscar-amezquita) [reply app/src/main/java/com/globant/example/mentorapp/view/activities/SplashScreenActivity.java]: It's true, lines deleted.

File: app/src/main/java/com/globant/example/mentorapp/view/activities/SplashScreenActivity.java (hunk 1 above)
Comment (oscar-amezquita) [reply app/src/main/java/com/globant/example/mentorapp/view/activities/SplashScreenActivity.java]: You're right. Added `getResources().getInteger(R.integer.splash_screen_duration)`.

File: build.gradle (hunk 2 above)
Comment (oscar-amezquita) [reply build.gradle]: Isn't necessary. Line deleted.

File: app/src/main/java/com/globant/example/mentorapp/view/activities/SplashScreenActivity.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
+package com.globant.example.mentorapp.view.activities;
```
Comment (daniellemariani) [inline app/src/main/java/com/globant/example/mentorapp/view/activities/SplashScreenActivity.java]: Let's keep package names in singular

Comment (oscar-amezquita) [reply app/src/main/java/com/globant/example/mentorapp/view/activities/SplashScreenActivity.java]: Ok, package name changed.

**Review by daniellemariani — APPROVED** [review]
LG

**Pull request merged**